enclave_mgmt/compile_models.py

```diff
@@ -124,6 +126,31 @@ class Config:
         extra = Extra.forbid
 
 
+class InputInteractiveBlock(BaseModel):
+    id: UUID
+    content_id: UUID
+    variant: str
+    content: str
+    prompt: str
+
+    class Config:
+        extra = Extra.forbid
+
+
+class ProblemSetProblem(BaseModel):
+    id: UUID
+    content_id: UUID
+    variant: str
+    pset_id: UUID
+    content: str
+    problem_type: str
```

Comment:
Minor: We can type this a bit more strictly if we want to catch unexpected values:

```python
    problem_type: Literal['input', 'dropdown', 'multiselect', 'multiplechoice']
```